be/src/Project/Services/Catalog/Catalog.Api/v1/Categories/Controllers/CategoriesController.cs

```diff
@@ -345,7 +343,7 @@ public async Task<IActionResult> SaveCategorySchema(CategorySchemaRequestModel r
         /// <param name="categoryId">The category id.</param>
         /// <returns>The category schema.</returns>
         [HttpGet, MapToApiVersion("1.0")]
-        [Route("CategorySchemas/{categoryId}")]
+        [Route("CategorySchema/{categoryId}")]
```

Comment:
**suggestion:** Best practices for naming endpoints for example https://restfulapi.net/resource-naming/: Change CategorySchema to CategorySchemas